langwatch/src/utils/dbMultiTenancyProtection.ts

```diff
@@ -12,6 +12,7 @@ const EXEMPT_MODELS = [
   "OrganizationInvite",
   "Project",
   "Subscription",
+  "Trigger",
```

Comment:
ooooooh that's why, you removed the protection on purpose, why? Delete this line